src/test/resources/net/sf/jabref/model/duplicates.bib

```diff
@@ -0,0 +1,17 @@
+% Encoding: UTF-8
+
+@InProceedings{dupJournalTechreport,
+  author    = {Kolb, Stefan and Wirtz, Guido},
+  title     = {This was published at a Workshop and a Journal with the same Title},
+  booktitle = {Proc. Workshop Cloud},
+  year      = {2016},
+}
+
+@Article{dupJournalTechreport,
+  author  = {Kolb, Stefan and Wirtz, Guido},
```

Comment:
Can we also add a test where the authors are written in a different serialization? Firstname Lastname?